cmd/pd-server/main.go

```diff
@@ -47,7 +49,20 @@ import (
 )
 
 func main() {
-	ctx, cancel, svr := createServerWrapper(os.Args[1:])
+	var ctx context.Context
+	var cancel context.CancelFunc
+	var svr basicsvr.Server
+
+	if len(os.Args) < 2 {
```

Comment:
BTW, we have synced offline. @lhy1024 will implement a more formal way to parse subcommand here.

Reply:
I'll remove  tso path for now and just keep  ctx, cancel, svr := createServerWrapper(os.Args[1:]) to integrate with @lhy1024's change mentioned above.